greengrass_ipc/source/GreengrassCoreIpcModel.cpp

```diff
@@ -5420,6 +5420,10 @@ namespace Aws
             {
                 payloadObject.WithString("versionStage", m_versionStage.value());
             }
+            if (m_refresh.has_value())
+            {
+                payloadObject.WithBool("refresh", m_refresh.value());
+            }
         }
 
         void GetSecretValueRequest::s_loadFromJsonView(
```

Comment:
Not happy that we're deserializing requests, but not much can be done about it at this point.